File: 240808/폭탄 해제/bomb-removal.cpp
#include <iostream>
#include <string>
using namespace std;

class My {
public: 
    string code;
    char color;
    int sec;

    My(string code, char color, int sec) {
        this -> code = code;
        this -> color = color;
        this -> sec = sec;
    }
};

int main() {

    string code;
    char color;
    int sec;

    cin >> code >> color >> sec;

    My a(code, color, sec);

    cout << "code : " << a.code << '\n' 
         << "color : " << a.color << '\n' 
         << "second : " << a.sec << '\n' ;


    return 0;
}

File: 241019/폭탄 해제/bomb-removal.py
class bomb :
    def __init__(self, code=0, color=0, sec=0) :
        self.code = code
        self.color = color
        self.sec = sec

b = bomb()
b.code, b.color, b.sec = tuple(input().split())

print(f"code : {b.code}")
print(f"color : {b.color}")
print(f"second : {b.sec}")